src/alg_utils.jl

```diff
@@ -70,6 +70,7 @@ alg_extrapolates(alg::Hairer4) = true
 alg_extrapolates(alg::Hairer42) = true
 alg_extrapolates(alg::IRKN4) = true
 alg_extrapolates(alg::IRKN3) = true
+alg_extrapolates(alg::ABDF2) = true
```

Comment:
This is good to add, but I don't think it actually extrapolates yet? The extrapolating initial guess is something that should be added in the future though.